### ISSUE TITLE
provide access to the time of an animation layer

### ISSUE THREAD

**Issue by stephengold** [jMonkeyEngine/jmonkeyengine#1200]
I want to synchronize some anim clips with other events occuring in my app.

The old system provides `AnimChannel.getTime()` and `.setTime()` to access the time of the animation that’s playing. In the new system, time is stored in the `Layer`; there’s no `getTime()` or `setTime()`.

Public getters and setters should be added, probably to the `AnimComposer` class.

For more information, see the forum topic: https://hub.jmonkeyengine.org/t/synchronizing-an-animclip/42393

**Comment by Ali-RS** [comment]
Is this Ok

```
    /**
     * Returns current time of the specified layer.
     */
    public double getTime(String layerName) {
        Layer l = layers.get(layerName);
        if (l == null) {
            throw new IllegalArgumentException("Unknown layer " + layerName);
        }
        return l.time;
    }

    /**
     * Set current time on the specified layer. 
     */
    public void setTime(String layerName, double time) {
        if (time < 0) {
            throw new IllegalArgumentException("Time can not be negative");
        }
        
        Layer l = layers.get(layerName);
        if (l == null) {
            throw new IllegalArgumentException("Unknown layer " + layerName);
        }
        l.time = time;
    }
```

**Comment by stephengold** [comment]
That's roughly what I have in mind. Note: lenght -> length.

**Comment by Ali-RS** [comment]
@stephengold  PR is submitted: https://github.com/jMonkeyEngine/jmonkeyengine/pull/1201

**Comment by stephengold** [comment]
Thank you!

**Comment by stephengold** [comment]
Resolved in `master` branch at 5c75af1d38f679c6a42f3ca02394a72e43c1ec34 .
